src/views/ModelPlan/TaskList/Payment/Complexity/index.tsx

```diff
@@ -0,0 +1,544 @@
+                            }
+                            onChange={() => {
+                              setFieldValue(
+                                'canParticipantsSelectBetweenPaymentMechanisms',
+                                false
+                              );
+                            }}
+                          />
+                        </Fieldset>
+                        <AddNote
+                          id="payment-multiple-payments-note"
+                          field="canParticipantsSelectBetweenPaymentMechanismsNote"
+                        />
+                      </FieldGroup>
+
+                      <FieldGroup
+                        scrollElement="payment-frequency-payments"
+                        error={!!flatErrors.anticipatedPaymentFrequency}
+                        className="margin-top-4"
+                      >
+                        <Label htmlFor="payment-frequency-payments">
+                          {t('anticipatedPaymentFrequency')}
+                        </Label>
+                        <FieldErrorMsg>
+                          {flatErrors.anticipatedPaymentFrequency}
+                        </FieldErrorMsg>
+
+                        <Field
+                          as={MultiSelect}
+                          id="payment-frequency-payments"
+                          name="payment-frequency-payments"
+                          options={mappedAnticipatedPaymentFrequencyType}
+                          selectedLabel={t(
+                            'selectedAnticipatedPaymentFrequency'
+                          )}
+                          onChange={(value: string[] | []) => {
+                            setFieldValue('anticipatedPaymentFrequency', value);
+                          }}
+                          initialValues={
+                            initialValues.anticipatedPaymentFrequency
+                          }
+                        />
+
+                        {(values?.anticipatedPaymentFrequency || []).includes(
+                          AnticipatedPaymentFrequencyType.OTHER
+                        ) && (
+                          <FieldGroup
+                            scrollElement="payment-frequency-payments-other"
+                            error={
+                              !!flatErrors.anticipatedPaymentFrequencyOther
+                            }
+                          >
+                            <Label
+                              htmlFor="anticipatedPaymentFrequencyOther"
+                              className="text-normal"
+                            >
+                              {t('selectClaimsOther')}
+                            </Label>
+                            <FieldErrorMsg>
+                              {flatErrors.anticipatedPaymentFrequencyOther}
+                            </FieldErrorMsg>
+                            <Field
+                              as={TextField}
```

Comment:
The component being passed here should be `TextInput`, as `TextField` results in an uncontrolled input type.